charts/tidb-cluster/values.yaml

```diff
@@ -336,6 +336,15 @@ monitor:
   persistent: false
   storageClassName: local-storage
   storage: 10Gi
+  initializer:
+    image: pingcap/tidb-monitor-initializer:v3.0.0
```

Comment:
This should be v3.0.1 to match the default tidb version.

Reply:
Done